src/PhpPact/Standalone/Runner/ProcessRunner.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace PhpPact\Standalone\Runner;
+
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\ProcessBuilder;
```

Comment:
It is removed since 4.0. But using namespace doesn't trigger any error.

Another problem is that despite `>=2.8 <4.1` version constraint for symfony/process composer still takes 3.3. And `composer why` doesn't explaij why. Same in master branch but probably there are still php-cs-fixer dependencies

Reply:
I can see this returning an error or warning in a later version of PHP.  What about using fully qualified names instead of `use` a namespace?